src/tests/ftest/io/eight_servers.py

```diff
@@ -26,112 +26,58 @@
 import os
 import json
 import distutils.spawn
-from apricot import Test
+from apricot import TestWithServers
 
-import agent_utils
-import server_utils
 import write_host_file
 import ior_utils
-from daos_api import DaosContext, DaosPool, DaosApiError
+from daos_api import DaosPool, DaosApiError
 
-class EightServers(Test):
+class EightServers(TestWithServers):
     """
     Test class Description: Runs IOR with 8 servers.
+    :avocado: recursive
     """
 
-    def __init__(self, *args, **kwargs):
-
-        super(EightServers, self).__init__(*args, **kwargs)
+    def setUp(self):
```

Comment:
I was just reviewing the SetUp in apricot and it looks like you don't need the setUp at all ...
The client hostfiles are also created in setUp in TestWithServer class

Reply:
Done